cli/utilities/utils.py

```diff
@@ -229,3 +230,62 @@ def verify_execution_status(out, cmd):
             log.error(f"Execution failed for '{cmd}' on '{node}'")
             return False
     return True
+
+
+def set_selinux_mode(nodes, enforcing_mode):
+    """
+    Sets the selinux mode to the specified value and validate whether the selinux mode is set
+    Args:
+        nodes (ceph): ceph node objects
+        enforcing_mode (str): enforcing/permissive/disabled
+    """
+    if enforcing_mode == "permissive":
+        mode = "0"
+    elif enforcing_mode == "enforcing":
```

Comment:
missing `disabled`  ?

Reply:
disabled cannot be done via setenforce if I'm not wrong. So removing that option from the method, To add this functionality we need to edit the etc/selinux/config file. If the requirement comes, this can be appened to the code. As of now the scenarios covered in polarion has only permissive and enforcing